pkg/kafka/cluster_admin_client_mock_impl.go

```diff
@@ -26,13 +26,26 @@ const (
 	defaultMockControllerID = 1
 )
 
-var (
-	// defaultMaxMessageBytes specifies the default max message bytes.
-	defaultMaxMessageBytes = "10485760"
-	// defaultMaxMessageBytes specifies the default min insync replicas for broker and default topic.
+const (
+	// defaultBrokerMessageMaxBytes specifies the broker's `message.max.bytes`, default to 1048576.
```

Comment:
Can you explain why this value was chosen?

Reply:
Since the maximum size of a single tidb txn is 6m, we picked a value larger than that.

Reply:
Since the maximum size of a single tidb txn is 6m, we picked a value larger than that.

Reply:
this value is the default value for kafka topic's `max.message.bytes`, and `message.max.bytes`, `min.insync.replicas`, identical to their default value .

for ticdc `MaxMessageBytes` is default to `10m` in another config file.

Reply:
I think it is good to add some comments.